pages/linux/xeyes.md

```diff
@@ -0,0 +1,11 @@
+# xeyes
+
+> Display eyes on the screen that follow the mouese cursor
+
+- Launch xeyes on local display:
+
+`xeyes`
+
+- Launch xeyes on a remote display:
+
+`xeyes -display {{remote.host}}:0.0`
```

Comment:
`0.0` means 

Terminal `0` / Screen `0`  

`:0.1` would be the second monitor

`:1` could be a VNC server